exercises/grains/big-integer.spec.js

```diff
@@ -1,59 +0,0 @@
-var BigInt = require('./big-integer');
```

Comment:
I don't think this file should be deleted. It serves as documentation for the `big-integer` library. If it contains ESLint errors, it could be ignored like `big-integer.js`